main/src/cgeo/geocaching/connector/lc/LCApi.java

```diff
@@ -67,10 +67,27 @@ protected static Geocache searchByGeocode(final String geocode) {
             return Collections.emptyList();
         }
 
-        final double latcenter = (viewport.getLatitudeMax()  + viewport.getLatitudeMin())  / 2;
-        final double loncenter = (viewport.getLongitudeMax() + viewport.getLongitudeMin()) / 2;
-        final Geopoint center = new Geopoint(latcenter, loncenter);
-        return searchByCenter(center);
+        final double lat1 = viewport.getLatitudeMax();
+        final double lat2 = viewport.getLatitudeMin();
+        final double lon1 = viewport.getLongitudeMax();
+        final double lon2 = viewport.getLongitudeMin();
+        final double latcenter = (lat1 + lat2) / 2;
+        final double loncenter = (lon1 + lon2) / 2;
+        final Geopoint gp1 = new Geopoint(lat1, lon1);
+        final Geopoint gp2 = new Geopoint(lat2, lon2);
+        final double radius = gp1.distanceTo(gp2) * 500; // we get diameter in km, need radius in m
+        Log.d("_LC Radius: " + String.valueOf((int) radius));
+        final Parameters params = new Parameters("skip", "0");
+        params.add("take", "500");
+        params.add("radiusMeters", String.valueOf((int) radius);
```

Comment:
A ")" is missing here...

Reply:
that happens if you're not at the android studio system and commit changes without doing a compile first.

I need to get my commandline gradle comand working. Perhaps that is the opportunity now to learn it. How do I trigger a build from the commandline?

Reply:
I think a `./gradlew build` should do the trick. But cannot say as I don't use it myself...